verifier/verifier.go

```diff
@@ -177,6 +178,18 @@ func (v *verifier) processSignature(ctx context.Context, sigBlob []byte, envelop
 			return err
 		}
 
+		pluginVersion := metadata.Version
+		var semVerRegEx = regexp.MustCompile(`^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`)
+
+		//checking if the plugin version is in valid semver format
+		if !semVerRegEx.MatchString(pluginVersion) {
+			return notation.ErrorVerificationInconclusive{Msg: fmt.Sprintf("plugin %s has pluginVersion %s which is not in valid semver format", verificationPluginName, pluginVersion)}
+		}
```

Comment:
will this throw an error when `verificationPluginMinVersion` field is not present in signature?

Reply:
you mean plugin version or minPluginVersion?

Reply:
Commented on wrong line added comment on line 190.

Reply:
Addressed in other comment